handlers/api-handler.ts

```diff
@@ -187,7 +187,7 @@ export function ApiHandler<
 				);
 			}
 
-			let queryBody: unknown;
+			let queryBody = {};
```

Comment:
Does this handle sending text as the body?

Reply:
It would not... Even before though it would not. This is because right after this line, it would go through a parser (`qs.parse('some string')`) which would return `{ 'some string': '' }` and fail validation.

Regardless, it's bad practice and I don't think faceteer should support that.

If the developer did want to support this, they would do a workaround by grabbing the raw query string from the event (`event.rawQueryString`)